modules/mvt/src/lib/parse-mvt.js

```diff
@@ -54,10 +54,11 @@ function getDecodedFeature(feature, options = {}, layerName) {
       : transformCoordinates(feature, transformToLocalCoordinates);
 
   // Add layer name to GeoJSON properties
-  const hasLayerProperty = decodedFeature.properties.hasOwnProperty('layer');
-  const layerPropertyName = hasLayerProperty ? options.layerProperty : 'layer';
-  if (layerPropertyName) {
-    decodedFeature.properties[layerPropertyName] = layerName;
+  const isLayerPropertyPresent = decodedFeature.properties.layer;
+  if (options.layerProperty) {
+    decodedFeature.properties[options.layerProperty] = options.layerName;
+  } else if (!isLayerPropertyPresent) {
+    decodedFeature.properties.layer = options.layerName;
```

Comment:
I think we decided `layerName` was going to be the default. https://github.com/uber-web/loaders.gl/issues/674#issuecomment-597344966

Reply:
Yes, sorry about that, I am just changing it now.